libsrc/apple2/get_ostype.s

```diff
@@ -19,6 +24,17 @@ initostype:
         tya
         ora     #$80
 done:   sta     ostype
+
+        .ifndef __APPLE2ENH__
+        cmp     #$30          ; Apple //e?
+        ror                   ; Carry to high bit
+        sta     iie_or_newer
+
+        bpl     :+            ; Set uppercase mask
+        lda     #$FF
+        sta     uppercasemask
```

Comment:
It seems to me it shouldn't be the task of get_ostype to set uppercasemask. I don't want get_ostype to become the "initialize everything we likely need somewhere" function.
I believe iie_or_newer - with a different name - should get it's own file. It should get its own constructor that runs after the get_ostype constructor. That constructor checks ostype and sets the - renamed - iie_or_newer accordingly.
In the same way I believe uppercasemask should stay in its own file. It should get its own constructor that runs after the get_ostype constructor. That constructor checks ostype and sets uppercasemask accordingly.